index.html

```diff
@@ -113,14 +146,17 @@ <h4 style="font-size: 20px;">Developer </h4>
             </ul>
  </body>
 <!--JS-->
+<br>
+<br>
```

Comment:
Why have u added `<br>` after the body has been closed

Reply:
to leave space between the blue bar

Reply:
You are putting it after the body has been close ... it will not imapct